README.md

```diff
@@ -99,15 +125,19 @@ Until **Jest v27** is published, you might need to patch `jest` in order to load
 2. Manually update `node_modules/jest-runtime/build/index.js` file and replicate [this commit](https://github.com/facebook/jest/commit/e5a84d92fc906a5bb140f9753b644319cea095da#diff-c0d5b59e96fdc7ffc98405e8afb46d525505bc7b1c24916b5c8482de5a186c00)
 3. Run `npx patch-package jest-runtime` or `yarn patch-package jest-runtime`
 
+## Examples
+
+Under [examples folder][examples-folder] we're documenting the testing cases which `next-page-tester` enables.
+
 ## Notes
 
 - Data fetching methods' context `req` and `res` objects are mocked with [node-mocks-http][node-mocks-http]
-- Next page tester can be used with any testing framework/library (not tied to Testing library)
+- Next page tester can be used with any testing framework/library (it's not tied to Jest or Testing library)
```

Comment:
Should we mention that library hasn't been tested with other test runners than `jest` and issues with those should be reported.

Reply:
✅